Trimble.Modus.Components/Constant/ColorsConstants.cs

```diff
@@ -1,32 +1,35 @@
-﻿namespace Trimble.Modus.Components.Constant;
-internal class ColorsConstants
-{
-    public const string TrimbleBlue = "TrimbleBlue";
-    public const string TrimbleBlueDark = "TrimbleBlueDark";
-    public const string TrimbleYellow = "TrimbleYellow";
-    public const string TrimbleGray = "TrimbleGray";
-    public const string TrimbleDisabledColor = "TrimbleDisabledColor";
-    public const string Black = "Black";
-    public const string White = "White";
-    public const string DangerRed = "DangerRed";
-    public const string SecondaryButton = "SecondaryButton";
-    public const string TertiaryButton = "TertiaryButton";
-    public const string GrayPlaceholder = "GrayPlaceholder";
-    public const string FocusBlue = "FocusBlue";
-    public const string TrimbleReadOnlyGray = "TrimbleReadOnlyGray";
-    public const string DangerRedClicked = "DangerRedClicked";
-    public const string SecondaryButtonClicked = "SecondaryButtonClicked";
-    public const string TertiaryButtonClicked = "TertiaryButtonClicked";
-    public const string TrimbleBlueClicked = "TrimbleBlueClicked";
-    public const string BluePale = "BluePale";
-    public const string NeutralGray = "NeutralGray";
-    public const string Green = "Green";
-    public const string Transparent = "Transparent";
-    public const string CardPressed = "CardPressed";
-    public const string DangerToastColor = "DangerToastColor";
-    public const string WarningToastColor = "WarningToastColor";
-    public const string SuccessToastColor = "SuccessToastColor";
-    public const string ToastTextBlue = "ToastTextBlue";
-    public const string ModalGraySemiTransparent = "ModalGraySemiTransparent";
-    public const string ModalShadowColor = "ModalShadowColor";
-}
+namespace Trimble.Modus.Components.Constant;
+internal class ColorsConstants
+{
+    public const string TrimbleBlue = "TrimbleBlue";
+    public const string TrimbleBlueDark = "TrimbleBlueDark";
+    public const string TrimbleYellow = "TrimbleYellow";
+    public const string TrimbleGray = "TrimbleGray";
+    public const string TrimbleDisabledColor = "TrimbleDisabledColor";
+    public const string Black = "Black";
+    public const string White = "White";
+    public const string DangerRed = "DangerRed";
+    public const string SecondaryButton = "SecondaryButton";
+    public const string TertiaryButton = "TertiaryButton";
+    public const string GrayPlaceholder = "GrayPlaceholder";
+    public const string FocusBlue = "FocusBlue";
+    public const string TrimbleReadOnlyGray = "TrimbleReadOnlyGray";
+    public const string DangerRedClicked = "DangerRedClicked";
+    public const string SecondaryButtonClicked = "SecondaryButtonClicked";
+    public const string TertiaryButtonClicked = "TertiaryButtonClicked";
+    public const string PrimaryFloatingButtonClicked = "PrimaryFloatingButtonClicked";
```

Comment:
Do we need this? General button and the floating button have the same color, right?

Reply:
Yes, since floating button is a separate control and secondary has different onClick color.